pkg/gadgets/prometheus/example.yaml

```diff
@@ -0,0 +1,54 @@
+metrics:
+  - name: executed_processes
+    type: counter
+    category: trace
+    gadget: exec
+    labels:
+      - namespace
+      - pod
+      - container
+    #selector:
+    # - "comm:cat"
+
+
+#  counters:
+#    - name: "mymetric_root_executions"
```

Comment:
nit: These examples needs to be updated.

Reply:
ah right. That file is not intended to be committed. I'll remove it.